Update in driver.py to make ts_avg_window Optional, works for both ts_avg_window specified in yaml or not (#183)
NOAA-CSL/MELODIES-MONET: docs/appendix/yaml.rst

     used for averaging and plotting. Options are 'time' for UTC or 'time_local' 
     for local time
   * **ts_avg_window:** This is for timeseries plots only. This is the averaging 

>      window applied to the data. No averaging done if not provided in the yaml file (ts_avg_window is Optional now). Averaging is done if a pandas 
>      resample rule (e.g., 'H' is hourly, 'D' is daily) is specified for ts_avg_window.

Reviewer: ```suggestion
     window applied to the data. No averaging done if not provided in the yaml file (i.e., ts_avg_window is optional). Averaging is done if a pandas 
     resample rule (e.g., 'H' is hourly, 'D' is daily) is specified.
```
